src/components/popover/popover.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { ReactElement, ReactNode, forwardRef } from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import * as PopoverPrimitive from '@radix-ui/react-popover';
+import { getTheme } from '../utils/styles';
+
+interface Props {
+  children: ReactNode;
+  content: ReactNode;
+  active: boolean;
+  padding?: 'medium' | 'small' | 'none';
+  coloring?: 'light' | 'dark' | 'warning' | 'error';
+  placement?: 'top' | 'bottom' | 'left' | 'right';
+  alignment?: 'center' | 'start' | 'end';
+  hasPointer?: boolean;
+  offsetFromAnchor?: number;
+  passthroughProps?: {
+    [key: string]: string | number | boolean;
+  };
+  onExit?: () => void;
+  getInitialFocus?: () => void;
+  clickOutsideCloses?: boolean;
+  escapeCloses?: boolean;
+  hideWhenAnchorIsOffscreen?: boolean;
+  allowPlacementAxisChange?: boolean;
+};
+
+/**
+ * Display a popover. The popover is positioned relative to an anchor element
+ */
+export default function Popover({
+  coloring = 'light',
+  placement = 'right',
+  alignment = 'center',
+  padding = 'medium',
+  active = false,
+  hasPointer = true,
+  hideWhenAnchorIsOffscreen = false,
+  allowPlacementAxisChange = true,
+  clickOutsideCloses = true,
+  escapeCloses = true,
+  getInitialFocus,
+  offsetFromAnchor,
+  passthroughProps,
+  onExit,
+  children,
+  content
+}: Props): ReactElement {
+  const { background, borderColor, color, fill } = getTheme(coloring);
+
+  const bodyClasses = classnames(
+    `${background} ${borderColor} ${color} border shadow-darken25 round`,
+    {
+      'px12 py12': padding === 'medium',
+      'px12 py6': padding === 'small',
+      [`border border-1 border--${borderColor}`]: borderColor
+    }
+  );
+
+  const Anchor = forwardRef<HTMLButtonElement>((props, ref) => {
+    return (
+      <PopoverPrimitive.Anchor asChild>
+        <span {...props} ref={ref}>
+          {children}
+        </span>
+      </PopoverPrimitive.Anchor>
+    );
+  });
+
+  const getContent = () => {
+    if (typeof content === 'function') {
+      return content();
+    } else {
+      return content;
+    }
```

Comment:
```suggestion
    if (typeof content === 'function') {
      return content();
    }
    return content;
```
The `else` seems unnecessary here